src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -177,14 +204,22 @@ const chatReportSelector = (report) =>
  */
 const reportActionsSelector = (reportActions) =>
     reportActions &&
-    _.map(reportActions, (reportAction) => ({
-        errors: lodashGet(reportAction, 'errors', []),
-        message: [
-            {
-                moderationDecision: {decision: lodashGet(reportAction, 'message[0].moderationDecision.decision')},
-            },
-        ],
-    }));
+    _.map(reportActions, (reportAction) => {
+        const {reportActionID, parentReportActionID, actionName, errors = []} = reportAction;
+        const decision = _.get(reportAction, 'message[0].moderationDecision.decision');
```

Comment:
Not a big deal but any reason for replacing lodashGet with underscore?
In JS, we prefer lodash to underscore.
In TS, we prefer native method to lodash/underscore.

Reply:
With that I generally try to follow what the rest of the file is doing. In this one, `_` was already imported and used throughout the file so I was following that. I can put back lodashGet.

Reply:
No need to put back. This component will be migrated to TS soon anyway

Reply:
heh too late.